packages/core/src/components/information-bar/information-bar.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Element, h, Host, Prop, Watch } from '@stencil/core';
+
+export type BarNumbers = {
+  count: number;
+  stripped: number;
+  icon: string;
+  color: string;
+};
+
+@Component({
+  tag: 'information-bar',
+  styleUrl: 'information-bar.scss',
+  shadow: true,
+})
+export class InformationBar {
+  @Element() el: HTMLInformationBarElement;
+
+  @Prop() bar: BarNumbers[] = [];
```

Comment:
Missing JSDoc comment for Prop decorated variables.

```suggestion
  /**
   * Configuration of the bar
   */
  @Prop() bar: BarNumbers[] = [];
```